solver/llbsolver/file/backend.go

```diff
@@ -380,19 +380,6 @@ func (fb *Backend) readUserWrapper(owner *pb.ChownOpt, user, group fileoptypes.M
 }
 
 func cleanPath(s string) (string, error) {
```

Comment:
This could have some more descriptive name.

Reply:
How about `cleanAndNormalizePath`?